packages/vite/src/node/ssr/fetchModule.ts

```diff
@@ -77,7 +83,8 @@ export async function fetchModule(
       throw err
     }
     const file = pathToFileURL(resolved.id).toString()
-    const type = isFilePathESM(file, server.config.packageCache)
+    // NOTE: fileURLだとキャッシュにヒットしないと思う
```

Comment:
👀

Reply:
Oops, I mixed up the part I was writing as a note to separate into a different PR. 😅